main.js

```diff
@@ -0,0 +1,83 @@
+//querySelectors
+
+var addRecipeBtn = document.querySelector('.recipe-btn');
+var letsCookBtn = document.querySelector('.cook-btn');
+
+//eventListeners
+//buttons
+//does the below button even need functionality?
+addRecipeBtn.addEventListener('click', addRecipe);
+//DO NEED
+//below to push out a food selection once users clicks
+letsCookBtn.addEventListener('click', letsCook)
+
+
+//object for meal? or individual foods? do i even need this?
+// maybe to push into full meal array for if i get to the CYOA? A full meal array?
+//would it update to output a full array?
+//do i need a class instead or at all?
+//i think i do because of there are several arrays running but can they just pass through the object?
+
+class Foods {
+  constructor (sides, mains, desserts, entireMeal) {
+    this.sides = sides;
+    this.mains = mains;
+    this.desserts = desserts;
+    //not sure if entireMeal can go into the constructor?
+    //this is possible going to be its own object instead?
+    //leave out of constructor ?
+    //should it have its own object instance instead? how to do?
+    this.entireMeal = entireMeal;
```

Comment:
from the [comp](https://frontend.turing.edu/projects/module-1/dinner.html), Don’t worry about the “Add a Recipe” button and “Entire Meal” radio button for now.